website/docs/plugins/rq_launcher.md

```diff
@@ -97,4 +97,6 @@ $ python my_app.py --multirun task=1,2,3,4,5
 [HYDRA] Polling job statuses every 1.0 sec
 ```
 
+Note that any third party library dependencies need to be installed on all nodes. For serialization of jobs [`cloudpickle`](https://github.com/cloudpickle/cloudpickle) is used.
```

Comment:
Correct me if I am wrong, but it's a bit more than this:
the dependencies needs to be installed in the Python interpreter used to run the RQ worker, right?

Reply:
Yes, how would you phrase it instead?

Reply:
Perhaps: 
> Note that any dependencies need to be installed in the Python environment used to run the RQ worker.

Reply:
Sounds good.

Did you rebase?

Reply:
Yes, I did: 
> This branch is 2 commits ahead of facebookresearch:master.

Not sure why tests are failing. I'll change the sentence to the one above. Hopefully they'll pass next time.